tlair/src/test/scala/at/forsyte/apalache/tla/lir/TestModule.scala

```diff
@@ -39,26 +39,24 @@ class TestModule extends AnyFunSuite {
      * ABInit = /\ msgQ = <<>> /\ ackQ = <<>> /\ sBit \in {0,1} /\ sAck = sBit /\ rBit = sBit /\ sent \in Data /\ rcvd
      * \in Data
      */
-    val ABInit =
-      new TlaOperDecl("ABInit", List(),
-          OperEx(TlaBoolOper.and, OperEx(TlaOper.eq, NameEx("msgQ"), emptySeq),
-              OperEx(TlaOper.eq, NameEx("ackQ"), emptySeq), OperEx(TlaSetOper.in, NameEx("sBit"), ZeroOneSet),
-              OperEx(TlaOper.eq, NameEx("sAck"), NameEx("sBit")), OperEx(TlaOper.eq, NameEx("rBit"), NameEx("sBit")),
-              OperEx(TlaSetOper.in, NameEx("sBit"), ZeroOneSet), OperEx(TlaSetOper.in, NameEx("sBit"), ZeroOneSet)))
+    new TlaOperDecl("ABInit", List(),
+        OperEx(TlaBoolOper.and, OperEx(TlaOper.eq, NameEx("msgQ"), emptySeq),
+            OperEx(TlaOper.eq, NameEx("ackQ"), emptySeq), OperEx(TlaSetOper.in, NameEx("sBit"), ZeroOneSet),
+            OperEx(TlaOper.eq, NameEx("sAck"), NameEx("sBit")), OperEx(TlaOper.eq, NameEx("rBit"), NameEx("sBit")),
+            OperEx(TlaSetOper.in, NameEx("sBit"), ZeroOneSet), OperEx(TlaSetOper.in, NameEx("sBit"), ZeroOneSet)))
```

Comment:
A fruitless value

Reply:
I will leave these for now, and instead open an issue to have Jure (who authored this test) check it out to make sure its testing whatever he'd intended it to.